### PR TITLE
feat(pdcli): add aliases

### PR DESCRIPTION
**Pull request by georgettica**
copied from @bng0y

### CODE REVIEW

File: utils/bashrc.d/10-pdcli.bashrc [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+# pdlo will login to the pd service
+alias pdlo='pd login'
+
+# pdsr will show resolved incidents for a service
+function pdsr() {
+  local service=$1
+  pd incident:list -s resolved -S $service
+}
+
+SILENT_TEST_USER_ID=P8QS6CC
```
Comment (clcollins) [inline utils/bashrc.d/10-pdcli.bashrc:14]: Might be a personal preference, but maybe group "global"-type variables at the top of the file for quick reference?

**Review by clcollins — COMMENTED** [review]
Overall looks good to me.  Added two small comments.

I'm sorta torn about adding aliases here for everyone rather than as a personal change, but at the same time,they can just not be used if desired. Love the work here though!

File: utils/bashrc.d/10-pdcli.bashrc [hunk 2]
```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+#set -eEuo pipefail
+
+# pdlo will login to the pd service
+alias pdlo='pd login'
```
[the remark this answers is not in the document]
Comment (georgettica) [reply utils/bashrc.d/10-pdcli.bashrc:6]: that's right. removing

File: utils/bashrc.d/10-pdcli.bashrc (hunk 1 above)
Comment (georgettica) [reply utils/bashrc.d/10-pdcli.bashrc:14]: yeah, I can do that, no problem :)

File: utils/bashrc.d/10-pdcli.bashrc [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+# pdilu will list incidents based on the user
+function pdilu() {
+  if [[ $# -eq 0 ]]; then
+    echo "cannot list pd incidents by user where no user is provided"
+    return
+  fi
+  user="$@"
+  
+  pd incident:list -e $user
```
[the remark this answers is not in the document]
Comment (georgettica) [reply utils/bashrc.d/10-pdcli.bashrc:59]: i'll try it :)

**Pull request merged**